charts/nextcloud/values.yaml

```diff
@@ -3,7 +3,9 @@
 ##
 image:
   repository: nextcloud
-  # tag: 24.0.3-apache
+  flavor: apache
```

Comment:
I remember in the past there was some back and forth about where the default flavor should be defined. I think having it in the values.yaml file makes the most sense.